Solutions/Recorded Future/Playbooks/RecordedFuture-DOMAIN-C2_DNS_Name-IndicatorProcessor.json

```diff
@@ -16,7 +16,8 @@
         }
     },
     "variables": {
-        "RecordedFutureConnectionName": "[concat('recordedfuture-', parameters('PlaybookName'))]"
+        "RecordedFutureConnectionName": "[concat('recordedfuture-', parameters('PlaybookName'))]",
+        "RecordedFutureImportPlaybookName": "RecordedFuture-ImportToSentinel"
```

Comment:
This value should be fetched from the parameters. Hardcoding this value will lead to issues where the name of this playbook was changed by the customer.